### PR TITLE
Handling error when perunPrincipal.user is null

### LINKED ISSUE #113: Handle not existing users login
* There is a possibility, that someone is trying to access Perun with an identity which is not in Perun. In that case, Perun returns `null` user in Perun principal. If that happens, the gui has to notify the users about that.

### PR DESCRIPTION
**Pull request by DanoFecko**
Resolves #113 

* sorry for double commit, it was a lot easier to keep to old one here

### CODE REVIEW

File: libs/perun/services/src/lib/init-auth.service.ts [hunk 1]
```diff
@@ -40,8 +43,12 @@ export class InitAuthService {
     return this.authzService.getPerunPrincipal()
       .toPromise()
       .then(perunPrincipal => {
-        this.storeService.setPerunPrincipal(perunPrincipal);
-        this.authResolver.init(perunPrincipal);
+        if (perunPrincipal.user === null) {
+          this.dialog.open(UserDontExistDialogComponent, {height: '80px'});
```
Comment (Vojtech-Sassmann) [inline libs/perun/services/src/lib/init-auth.service.ts:47]: Why is there a height limit? This height causes a scrollbar to appear.

Comment (DanoFecko) [reply libs/perun/services/src/lib/init-auth.service.ts:47]: without it, dialog had lot of empty space under text. It didn't show scrollbar on my screen, but i didn't check smaller resolutions.

Comment (DanoFecko) [reply libs/perun/services/src/lib/init-auth.service.ts:47]: i will play a bit with it and find solution

Comment (Vojtech-Sassmann) [reply libs/perun/services/src/lib/init-auth.service.ts:47]: To reduce the padding or margin of an element, just change the CSS. Or, you can even create a new class with only those CSS attributes that you want to use.

Comment (Vojtech-Sassmann) [reply libs/perun/services/src/lib/init-auth.service.ts:47]: This should not be hard at all. It is just a matter of a couple of minutes to write a CSS class and apply it.

**Comment by DanoFecko** [comment]
@Vojtech-Sassmann I fixed by placing the text to the dialog content instead of title. I also disabled closing the dialog by clicking on the background and somehow I had .css file in component instead of .scss, so I changed that too.

**Pull request merged**